poc_iot_verifier/src/entropy.rs

```diff
@@ -2,6 +2,13 @@ use crate::{Error, Result};
 use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
 
+/// measurement in seconds of a piece of entropy
+/// its lifespan will be valid from entropy.timestamp to entropy.timestamp + ENTROPY_LIFESPAN
+/// any beacon or witness report received after this period and before the ENTROPY_STALE_PERIOD
+/// defined below will be rejected due to being outside of the entropy lifespan
+/// TODO: determine a sane value here
+pub const ENTROPY_LIFESPAN: i32 = 90;
```

Comment:
nit: Would be good to rename this to `ENTROPY_LIFESPAN_SECS`

Reply:
madninja not like that convention, it came up on another PR.